impls/src/adapters/emoji.rs

```diff
@@ -13,9 +13,9 @@
 // limitations under the License.
 
 use bitvec::prelude::*;
-use chrono::format::format;
+//use chrono::format::format; use in future
```

Comment:
If it's not being used you can remove it

Reply:
Makes sense, I'll delete